### PR TITLE
Changed dependency from yast2-mail to yast2 (FATE#319711)

### PR DESCRIPTION
**Pull request by teclator**
- Depend on yast/yast-yast2#450

### CODE REVIEW

File: package/yast2-security.changes [hunk 1]
```diff
@@ -4,6 +4,7 @@ Mon Mar  7 09:51:41 UTC 2016 - knut.anderssen@suse.com
 - Added support for multiple display managers (bnc#946889).
 - Replaced testsuite tests by rspec tests.
 - Removed autotools.
+- Updated yast-yast2 dependency for cfg_mail.scr
```
Comment (imobach) [inline package/yast2-security.changes]: Please, use `yast2` instead of `yast`.

File: package/yast2-security.spec [hunk 2]
```diff
@@ -27,19 +27,20 @@ Group:          System/YaST
...
 # new Pam.ycp API
 Requires:       yast2-pam >= 2.14.0
 
 # etc_sysctl_conf.scr
 # Wizard::SetDesktopTitleAndIcon
-# Directory.find_data_file
-Requires:       yast2 >= yast2-3.1.131
+# Directory.find_data_file >= yast2-3.1.131
+# cfg_mail.scr
+Requires:       yast2 >= yast2-3.1.179
```
Comment (jreidinger) [inline package/yast2-security.spec]: wrong format, it should be `yast2 >= 3.1.179`

File: package/yast2-security.spec [hunk 3]
```diff
@@ -27,19 +27,20 @@ Group:          System/YaST
...
 BuildRequires:  doxygen pkg-config perl-XML-Writer update-desktop-files yast2-pam
 BuildRequires:  yast2-devtools >= 3.1.10
-BuildRequires:  yast2-mail
 BuildRequires:  rubygem(yast-rake) >= 0.2.5
 BuildRequires:  rubygem(rspec)
-# Directory.find_data_file
-BuildRequires:  yast2 >= yast2-3.1.131
+# Directory.find_data_file >= yast2-3.1.131
+# cfg_mail.scr
+BuildRequires:  yast2 >= yast2-3.1.179
```
Comment (jreidinger) [inline package/yast2-security.spec]: same here

Comment (teclator) [inline package/yast2-security.spec]: Fixed. Thnx i didn't realize that there was a format error, just changed last digit.

File: package/yast2-security.spec [hunk 4]
```diff
@@ -27,19 +27,20 @@ Group:          System/YaST
...
+BuildRequires:  yast2 >= 3.1.179
 
 # new Pam.ycp API
 Requires:       yast2-pam >= 2.14.0
 
 # etc_sysctl_conf.scr
 # Wizard::SetDesktopTitleAndIcon
-# Directory.find_data_file
-Requires:       yast2 >= yast2-3.1.131
+# Directory.find_data_file yast2 >= 3.1.131
```
Comment (jreidinger) [inline package/yast2-security.spec]: this can be also removed. We also keep the latest reason why we need at least that version.

**Comment by jreidinger** [comment]
LGTM

**Pull request merged**